cmd/collectors/zapi/plugins/svm/svm.go

```diff
@@ -55,6 +56,11 @@ type cifsSecurity struct {
 	smbSigning      string
 }
 
+type ciphers struct {
```

Comment:
maybe a better named struct would be this?
```go
type sshInfo struct {
	ciphers     string
	insecured string or isInsecure
}
```

Reply:
added